app/views/complaints/_complaints_table_detail.html.erb

```diff
@@ -1,22 +1,23 @@
+<% complaint = Complaint.new(complaint_data) %>
+
 <tr>
-  <% formatted_status = status(complaint[:attributes]) %>
   <th scope="row" role="rowheader">
-    <span class="<%= formatted_status == "New" ? "text-bold" : "" %>">
-      <%= link_to complaint[:id], complaint_path(complaint[:id]), class: "usa-link" %>
+    <span class="<%= complaint.new? ? "text-bold" : "" %>">
+      <%= link_to complaint.id, complaint_path(complaint.id), class: "usa-link" %>
     </span>
   </th>
   <td
-    data-sort-value="<%= DateTime.parse(complaint[:attributes][:creationDate]).to_i %>"
+    data-sort-value="<%= sort_date(complaint.creation_date) %>"
   >
-    <%= DateTime.parse(complaint[:attributes][:creationDate]).strftime("%m/%d/%Y") %>
+    <%= complaint.formatted_creation_date %>
   </td>
   <td
-    data-sort-value="<%= ComplaintsHelper::STATUS_SORT_ORDER[formatted_status] %>"
+    data-sort-value="<%= sort_status(complaint.status) %>"
   >
-    <%= formatted_status %>
+    <%= complaint.status %>
   </td>
-  <td><%= complaint[:attributes][:grantee] %></td>
-  <td><%= truncate(complaint[:attributes][:summary]) %></td>
-  <td> TBD </td>
-  <td> TBD </td>
+  <td><%= complaint.grantee %></td>
+  <td><%= truncate(complaint.summary) %></td>
+  <td><%= "Yes" if complaint.has_tta_report? %></td>
```

Comment:
[question] should it have No if it doesn't have a report?

Reply:
@lauranash18f and I decided not to have anything about that, because if there isn't an activity report, say, for a complaint, then we have no way of actually knowing it doesn't exist vs only knowing that nothing has been linked.  In the original design, it had No / TBD / Yes but realistically all we can mark is yes unless if we simply mean that "no" means "nothing linked as far as the complaint tracker is concerned"